packages/react/src/__tests__/ReactProfiler-test.internal.js

```diff
@@ -627,15 +627,14 @@ describe('Profiler', () => {
 
         // Interrupt with higher priority work.
         // The interrupted work simulates an additional 5ms of time.
-        expect(
-          renderer.unstable_flushSync(() => {
-            renderer.update(
-              <React.unstable_Profiler id="test" onRender={callback}>
-                <Yield renderTime={5} />
-              </React.unstable_Profiler>,
-            );
-          }),
-        ).toEqual(['Yield:5']);
+        renderer.unstable_flushSync(() => {
+          renderer.update(
+            <React.unstable_Profiler id="test" onRender={callback}>
+              <Yield renderTime={5} />
+            </React.unstable_Profiler>,
+          );
+        });
+        expect(ReactTestRenderer.unstable_clearYields()).toEqual(['Yield:5']);
```

Comment:
Would be nice if this worked like the flush* methods– accepting an array of expected yields and auto-validating.

Not sure how that would work for `unstable_flushSync` though since that also uses `clearYields`. Maybe we just _shouldn't_ use `clearYields` in that method?